tests/test_gso.py

```diff
@@ -11,9 +11,9 @@
 
 
 if sys.maxsize >= 2**62:
-    dimensions = ((0, 0), (2, 2), (3, 3), (10, 10), (30, 30), (50, 50), (60, 60))
+    dimensions = ((1, 1), (2, 2), (3, 3), (10, 10), (30, 30), (50, 50), (60, 60))
```

Comment:
I think we should check these, those can happen as corner cases of higher-level algorithms

Reply:
Fair enough, I have changed matrix generation in this test file to use `bits=max(1, m)` in that case, as `bits=0` gave a Segmentation Fault.